### PR TITLE
Update to the latest pom and fix illegal access

### PR DESCRIPTION
**Pull request by res0nance**
See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

Migrating to the newest pom and removing illegal access should unblock #127 

### Proposed changelog entries

* Entry 1: Issue, Human-readable Text
* ...

### Submitter checklist

- [ ] JIRA issue is well described
- [ ] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change).
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [ ] Appropriate autotests or explanation to why this change has no tests

@fcojfernandez

### CODE REVIEW

**Comment by res0nance** [comment]
Incrementals not published due to [INFRA-2379](https://issues.jenkins-ci.org/browse/INFRA-2379)

**Comment by casz** [comment]
blocking #141

File: pom.xml [hunk 1]
```diff
@@ -19,6 +19,8 @@
   </description>
   <url>https://github.com/jenkinsci/cloudbees-folder-plugin/</url>
   <properties>
+    <revision>6.10.2</revision>
+    <changelist>-SNAPSHOT</changelist>
     <jenkins.version>2.130</jenkins.version>
```
Comment (jglick) [inline pom.xml]: Generally should bump to the next LTS (at least; at this point really 2.150.x is the oldest supportable baseline).
```suggestion
    <jenkins.version>2.150.3</jenkins.version>
```

File: src/main/resources/com/cloudbees/hudson/plugins/folder/views/Messages.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+Hudson.ViewName=All
```
Comment (jglick) [inline src/main/resources/com/cloudbees/hudson/plugins/folder/views/Messages.properties]:
```suggestion
Hudson.ViewName=All
```

(trailing newlines please)

**Pull request merged**